modules/op-starting-pipelines-from-topology-view.adoc

```diff
@@ -0,0 +1,36 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="op-starting-pipelines_from_topology_view{context}"]
+= Starting pipelines from Topology view
+
+For pipelines created using the *From Git* option, you can use the *Topology* view to interact with pipelines after you start them:
+
+[NOTE]
+====
+To see the pipelines created using the *Pipeline Builder* in the *Topology* view, customize the pipeline labels to link the pipeline with the application workload.
+====
+.Procedure
+
+To start a pipeline in *Topology* view:
+
+. On the left navigation panel, click *Topology*, and click on the application to see the pipeline runs listed in the side panel.
+. In the *Pipeline Runs* section, click *Start Last Run* to start a new pipeline run with the same parameters and resources as the previous one. This option is disabled if a pipeline run has not been initiated. You can also opt-in to starting the pipeline during the creation process itself.
++
+.Pipelines in Topology view
```

Comment:
Is this the beginning of a separate conceptual module?

Reply:
It's just a procedure which has been refactored inside a new one.

Reply:
@rolfedh This adds an image caption along with the figure number. Here is the preview: https://deploy-preview-42871--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/working-with-pipelines-using-the-developer-perspective#op-starting-pipelines_from_topology_viewusing-custom-pipeline-template-for-git-import